src/openshift/cluster.ts

```diff
@@ -73,9 +73,7 @@ export class Cluster extends OpenShiftItem {
             ignoreFocusOut: true,
             prompt: "Provide URL of the cluster to connect",
             validateInput: (value: string) => {
-                if (!validator.isURL(value)) {
-                    return 'Invalid URL provided';
-                }
+                if (!validator.isURL(value)) return 'Invalid URL provided';
```

Comment:
Implement it the same way you did with emptyName. Move this method to `OpenShiftItem` and call it here as 
``` TypeScript
validateInput: (value: string) => Cluster.isURL('Invalid URL provided', value)
```